### ISSUE TITLE
Freezing in the "Loading Terrain" when entering The OuterLands

### ISSUE THREAD

**Issue by sarboleda609** [Azanor/thaumcraft#1880]
Thaumcraft 4.2.3.5
Forge 1.7.10-1448

In my multiplayer and  singleplayer worlds (sp world is a copy of the mp one), I have this problem when entering the OuterLands in the eldritch obelisk close to my home. As soon as I teleport to it, it gets stuck in the "Loading terrain" thing forever (and doesn't respond to any of my commands if I try to stop it).
By using a copy of this world, I have tried different ways to fix this without sucess (at the end I was just trying everything I thought it would work...):
1. Redoing the portal by using the creative-only eldritch obelisk and using a new sinester node.
2. Finding another eldritch obelisk and create the portal. This allows me to enter to the outerLands but as soon as I try to use mine it freezes again and makes the second portal unresponsive too.
3.  Deleting region folder in /world/DIM-42
4. Step 3 then step 1 or 2
5. Deleting both region folder and labyrinth.dat file
6. Step 5 then step 1 or 2

After all of this, I decided to use an older backup I have of my mp server and by following step 5 and step 2, I was finally able to use the obelisk in my home. However, I have tried to replace every possible file (region folder in DIM42, labyrinth.dat, the chunk file of the overworld where the obelisks are) and I just cannot fix the copy of my multiplayer world.
Im kind of desperate so please help me out...

**Comment by cabralRodrigo** [comment]
@sarboleda609 Do you have the crash log?

**Comment by sarboleda609** [comment]
Nop. The multiplayer server just stops working overall (no logs, no nothing) and then I lose the connection. Same in the sp world.

**Comment by cabralRodrigo** [comment]
@sarboleda609 But you're crashing on a sp world, you have the crash log.

**Comment by sarboleda609** [comment]
Oh sorry, I thought I corrected the post but it seems it didn't change. It doesn't crash but freezes in the "Loading Terrain" thing forever. Im going to fix the post.

**Comment by cabralRodrigo** [comment]
@sarboleda609 So I think the FML Log could be helpful.

**Comment by sarboleda609** [comment]
The longest log ever...
https://gist.github.com/sarboleda609/210687f539d943656c1f

**Comment by cabralRodrigo** [comment]
@sarboleda609 You can reproduce this issue with only thaumcraft installed?

**Comment by sarboleda609** [comment]
Sorry for the delayed response. After downloading the world from my server (like 5 GBs..) and running it with only thaumcraft, it still has the same problem. I then tried to recreate the world, went to the obelisk near my home and I was able to enter the OuterLand dimension without any problem
Looks like we have some kid of corruption in our specific world.
My next idea is to copy the specific chunk where this obelisk is from the recreated world to the server one and try again...
